addon/mixins/in-viewport.js

```diff
@@ -3,53 +3,62 @@
 
 import Ember from 'ember';
 
-var on           = Ember.on;
-var get          = Ember.get;
-var Mixin        = Ember.Mixin;
-var debounce     = Ember.run.debounce;
+var on = Ember.on;
```

Comment:
this PR is very hard to read b/c you change the formatting. do you mind leaving the formatting as it was?